apps/api_web/lib/api_web/router.ex

```diff
@@ -23,14 +23,24 @@ defmodule ApiWeb.Router do
       Plug.Session,
       store: :cookie,
       key: "_api_key",
-      signing_salt: Application.get_env(:api_web, :signing_salt)
+      signing_salt: {Application, :get_env, [:api_web, :signing_salt]}
     )
 
+    @content_security_policy Enum.join(
+                               [
+                                 "default-src 'none'",
+                                 "img-src 'self' cdn.mbta.com",
+                                 "style-src 'self' 'unsafe-inline' maxcdn.bootstrapcdn.com fonts.googleapis.com",
+                                 "script-src 'self' maxcdn.bootstrapcdn.com code.jquery.com",
+                                 "font-src fonts.gstatic.com maxcdn.bootstrapcdn.com"
+                               ],
+                               "; "
+                             )
```

Comment:
Do we have a list of these other origins that need to be allowed or is this off the top of your head/from digging through the source?

Reply:
A combination of digging through the source and trying it out in my browser to see what was rejected by Firefox.